src/indexApi.ts

```diff
@@ -374,6 +374,30 @@ function checkForKeys(data: any): void {
     )
     throw new Error('Log includes sensitive data')
   }
+
+  const regex = /(\b[a-z]+\b\s){11,23}\b[a-z]+\b/g
+  const matches = dataString.match(regex)
+  const found = matches != null
+  if (found) {
+    for (const match of matches) {
+      const words = match.split(' ')
+      let allValid = true
+      for (const word of words) {
+        if (!wordlists.english.includes(word)) {
```

Comment:
Since there are 2048 words, should we turn this into a map for faster lookup time?

Reply:
done